modal-go/sandbox.go

```diff
@@ -214,6 +218,39 @@ func (sb *Sandbox) Tunnels(timeout time.Duration) (map[int]*Tunnel, error) {
 	return sb.tunnels, nil
 }
 
+// Poll checks if the Sandbox has finished running.
+// Returns nil if the Sandbox is still running, else returns the exit code.
+func (sb *Sandbox) Poll() (*int32, error) {
```

Comment:
To be slightly more ergonomic to the rest of go, should these codes be converted into `int`? Error codes go up to 255, so it'll fit.